ui/packages/components/src/Modal/Modal.tsx

```diff
@@ -82,3 +76,38 @@ export function Modal({
     </Dialog.Root>
   );
 }
+
+function Content({ children }: React.PropsWithChildren<{}>) {
+  return <div className="m-6">{children}</div>;
+}
+
+function Footer({ children, className }: React.PropsWithChildren<{ className?: string }>) {
+  return (
+    <div className={cn('border-t border-slate-200 p-6 dark:border-slate-800', className)}>
+      {children}
+    </div>
+  );
+}
+
+function Title({
```

Comment:
I wonder if this would be clearer if it was called Header, since it contains both the title and the description 🤔

Reply:
Good idea! I'll make that change